openrpc.json

```diff
@@ -1293,11 +1294,9 @@
               "$ref": "#/components/schemas/Integer"
             },
             "removed": {
-              "schema": {
-                "title": "logIsRemoved",
-                "description": "Whether or not the log was orphaned off the main chain",
-                "type": "boolean"
-              }
+              "title": "logIsRemoved",
+              "description": "Whether or not the log was orphaned off the main chain",
+              "type": "boolean"
```

Comment:
This may be a silly question... but should `boolean` get it's own components.schema and be used as a `$ref`? 
 
For example, `FilterId` gets one, but has no qualifiers (just a "fancy" name for a _string_). 
https://github.com/etclabscore/ethereum-json-rpc-specification/blob/0316db64e99fc72d69a21b6c72e1941fd79385d3/openrpc.json#L1115
(Or may FilterId) is just incomplete... waiting for a `pattern`?

So does `"Null"` https://github.com/etclabscore/ethereum-json-rpc-specification/blob/0316db64e99fc72d69a21b6c72e1941fd79385d3/openrpc.json#L1568

These are just questions, and I think not related directly to the changes presented. So from review standpoint, LGTM.

Reply:
Thank you for pointing this out. 

I agree in 97% of cases on this general sentiment. 

***To give some more context:** When you add a description field to the schema, it is going to be seen and treated by tooling as a different schema than the one its referencing, since it has different documentation. This gets very rabbit hole-y so ill leave it at that for now.*

There are a lot of cases in this OpenRPC document where the descriptions are pretty useless. 

I'd say that any PR removing descriptions and titles, such that the schema includes only a ref, would be valid topic for discussion / debate over the usefulness of the descriptions / titles ;)

Reply:
also, the whole `Null` thing was initially there as a "DRY" mechanism.... not sure its 100% still relevant either. ahaha.

Reply:
Roger, thanks, got it.